docs/pages/database-access/rbac/configuring-auto-user-provisioning.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: Database Automatic User Provisioning (Preview)
+description: Configure automatic user provisioning for databases.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
```

Comment:
We haven't updated all instance yet, but with the new transformer we only need an `opened` key, no value.

```suggestion
  opened
```